README.md

```diff
@@ -106,6 +106,34 @@ item.created? # => true
 item.selected? # => false
 ```
 
+### Filter methods
+
+Plugin can define dataset methods for all enum values:
+
+```ruby
+Item.plugin :enum_values, filter_methods: true # default is `false`
```

Comment:
OK, I agree, what about such changes then?

```suggestion
### Dataset filter methods

Plugin can define dataset filter methods for all enum values:

```ruby
Item.plugin :enum_values, filter_methods: true # default is `false`
```